app/org/sagebionetworks/bridge/models/schedules/CompoundActivityDefinition.java

```diff
@@ -0,0 +1,56 @@
+package org.sagebionetworks.bridge.models.schedules;
+
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.ser.impl.SimpleBeanPropertyFilter;
+import com.fasterxml.jackson.databind.ser.impl.SimpleFilterProvider;
+
+import org.sagebionetworks.bridge.dynamodb.DynamoCompoundActivityDefinition;
+import org.sagebionetworks.bridge.json.BridgeObjectMapper;
+import org.sagebionetworks.bridge.json.BridgeTypeName;
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+/**
+ * Compound activity definition, which is keyed off taskID and stored in persistent storage. This allows study
+ * developers to define a task once and use that task in multiple schedules.
+ */
+@BridgeTypeName("CompoundActivityDefinition")
```

Comment:
As mentioned in my other comment, we currently have CompoundActivity as the reference/instance object in the schedule, and CompoundActivityDefinition as the definition. We can change the names, but I don't think it would be any shorter.